# Goal definition
float32 x_pos
float32 y_pos
float32 yaw
--- 
# Result definition
bool position_reached
--- 
# Feedback definition
string current_status